signer/src/storage/model.rs

```diff
@@ -1034,7 +1034,7 @@ impl PartialOrd for StacksPrincipal {
 
 /// A ScriptPubkey of a UTXO.
 #[derive(Debug, Clone, Hash, PartialEq, Eq, PartialOrd, Ord)]
-pub struct ScriptPubKey(bitcoin::ScriptBuf);
+pub struct ScriptPubKey(pub bitcoin::ScriptBuf);
```

Comment:
I don't think that we should make this public. We should have what we need with our `Deref` and `From` implementations.